homeassistant/components/overkiz/water_heater_entities/atlantic_dhw.py

```diff
@@ -0,0 +1,177 @@
+"""Support for Atlantic Pass APC DHW."""
+
+from typing import Any, cast
+
+from pyoverkiz.enums import OverkizCommand, OverkizCommandParam, OverkizState
+
+from homeassistant.components.water_heater import (
+    STATE_ECO,
+    STATE_OFF,
+    STATE_PERFORMANCE,
+    WaterHeaterEntity,
+    WaterHeaterEntityFeature,
+)
+from homeassistant.const import ATTR_TEMPERATURE, UnitOfTemperature
+
+from .. import OverkizDataUpdateCoordinator
+from ..entity import OverkizEntity
+
+OVERKIZ_TO_OPERATION_MODE: dict[str, str] = {
+    STATE_PERFORMANCE: OverkizCommandParam.PERFORMANCE,
+    STATE_ECO: OverkizCommandParam.MANUAL_ECO_ACTIVE,
+    OverkizCommandParam.MANUAL: OverkizCommandParam.MANUAL_ECO_INACTIVE,
+}
+
+
+class AtlanticDomesticHotWaterProductionMBLComponent(OverkizEntity, WaterHeaterEntity):
+    """Representation of AtlanticDomesticHotWaterProductionMBLComponent (modbuslink)."""
+
+    _attr_temperature_unit = UnitOfTemperature.CELSIUS
+    _attr_supported_features = (
+        WaterHeaterEntityFeature.TARGET_TEMPERATURE
+        | WaterHeaterEntityFeature.OPERATION_MODE
+        | WaterHeaterEntityFeature.AWAY_MODE
+        | WaterHeaterEntityFeature.ON_OFF
+    )
+    _attr_operation_list = [*OVERKIZ_TO_OPERATION_MODE.keys()]
```

Comment:
You can just build the array here. No need to loop through the OVERKIZ_TO_OPERATION_MODE dict, when you are not using this dict.

Reply:
You are right. I am now working on it.

Please look at `"""Representation of AtlanticDomesticHotWaterProductionMBLComponent (modbuslink)."""` here. I don't know how it happened to be outdated in your comment above :)